### DISCUSSION TITLE
Зависание far2l после попытки копирования

### DISCUSSION THREAD

**Discussion by shmuz** [elfmz/far2l#1250] [Q&A]
С помощью F5 пытаюсь скопировать `~/.config/far2l/plugins/state.ini` (или что угодно другое из этой папки)
в `~/far2l/_build/install/Plugins`.
Фар зависает, приходится убивать его через System Monitor.
Это так должно быть? Если да, есть ли способ разморозить Фар, не убивая его процесс?

**Comment by shmuz** [comment]
Дополнительно, но возможно связано.
1. Становлюсь курсорной доской на какую либо папку из `~/.config/far2l/plugins`, например на `align`.
2. Нажимаю F8.
3. Без вывода диалога подтверждения удаления Фар сразу падает (исчезает).

**Update: нажатие F8 на чём угодно вызывает падение.**

**Reply by elfmz** [reply]
смотрите хвост обсуждения https://github.com/elfmz/far2l/issues/1248

**Comment by unxed** [comment]
Так чем тут закончилось в итоге? Так и падает?

**Reply by shmuz** [reply]
Починили.

**Reply by unxed** [reply]
И зависание и падение?

**Comment by shmuz** [comment]
Да.
